pkg/pillar/nireconciler/linux_currentstate.go

```diff
@@ -22,93 +22,91 @@ import (
 
 // Refresh the state of external items inside the Global subgraph of the current
 // state depgraph.
-func (r *LinuxNIReconciler) updateCurrentGlobalState(uplinksOnly bool) (changed bool) {
+func (r *LinuxNIReconciler) updateCurrentGlobalState(portsOnly bool) (changed bool) {
```

Comment:
I don't find portsOnly a very clear name here.
Based on the logic it seems to mean "do not include ACLs". 
Would it be more clear if you inverted the boolean and called it something like includeACLs?

Reply:
I changed it to `onlyPortsChanged bool` to make it clearer that only state of ports changed and requires update.
I don't want to do the opposite, because in the future there could be something additional to ACLs, which would not require update if only port state changed.

Reply:
Ok